src/views/ToDoList/TaskCard/index.tsx

```diff
@@ -0,0 +1,104 @@
+import {Link} from "react-router-dom";
+import ArrowLeftIcon from '@mui/icons-material/ArrowLeft';
+import HighlightOffIcon from '@mui/icons-material/HighlightOff';
+import ArrowRightIcon from '@mui/icons-material/ArrowRight';
+import AddIcon from '@mui/icons-material/Add';
+import { grey } from '@mui/material/colors';
+import {
+    TaskCardContainer,
+    TopBar,
+    TypographyLineLonger,
+    ButtonBar,
+    ButtonPreview,
+    TaskText
+} from "./TaskCard.styled.tsx";
+import {FunctionComponent} from "react";
+import {Task} from "../index.tsx";
+
+type TaskCardProps = {
+    task: Task,
+    fetchTasks(): any;
+}
+
+export const TaskCard: FunctionComponent<TaskCardProps> = (props) => {
+    const task = props.task
+
+    // @ts-ignore
+    async function removeTask(task) {
```

Comment:
Provide a type for the `task` argument to avoid the `implicitly has an 'any' type.` error